packages/code-du-travail-frontend/src/layout/Footer.js

```diff
@@ -26,19 +26,17 @@ const Footer = () => (
       <ServiceSection>
         <Container>
           <Title as="h3" topStripped>
-            Besoin d’un accompagnement personnalisé ?
+            Besoin de plus d’information ?
```

Comment:
on pourrait rajouter un `&nbps;`?

Reply:
of course. Par contrw j'arrive pas à faire passer l'enrigstrememt de l'image eleastic sur gitlab ca me rend ouf